package.json

```diff
@@ -13,7 +13,8 @@
     "eslint": "eslint src",
     "build": "babel src -d dist",
     "start": "node -- dist/index.js",
-    "deploy-now": "now && now alias"
+    "deploy-now": "now && now alias",
+    "deploy-dev": "now `sed 's/#.*$//;/^\\s*$/d;s/^/-e /' .env`"
```

Comment:
What is the purpose of deploy-dev?

Reply:
Deploys to now using the .env file instead of now.json file. .env has configuration for alzabottest app.